docs/application/dotnet/guides/nui/resources.md

```diff
@@ -61,26 +90,26 @@ Visual creation automatically determines whether the image is N_PATCH, SVG, or G
 | `CacheSize`   | Number of images to keep cached, can increase or decrease depending on memory available. |
```

Comment:
Add word "the"

Number of images to keep cached, can increase or decrease depending on the memory available.